src/KOKKOS/kokkos.h

```diff
@@ -32,6 +32,7 @@ class KokkosLMP : protected Pointers {
   int num_threads,ngpu;
   int numa;
   int auto_sync;
+  int gb_test;
```

Comment:
In all my tests, the default was fastest, so the flag can be eliminated, but it won't work for CUDA, so we'd need to keep a `#ifdef`.